packs/google/requirements.txt

```diff
@@ -0,0 +1 @@
+-e git+https://github.com/Kami/Google-Search-API.git@stackstorm_patched#egg=Google-Search-API
```

Comment:
Umm. Noticed this in urbandict pack too. Is it just some APIs on top of requests?